readme.md

```diff
@@ -381,6 +381,7 @@
 - [PowerShell](https://github.com/janikvonrotz/awesome-powershell#readme) - Cross-platform object-oriented shell.
 - [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows#readme) - Productivity app for macOS.
 - [Terminals Are Sexy](https://github.com/k4m4/terminals-are-sexy#readme)
+- [GitHub Actions](https://github.com/sdras/awesome-actions#readme)
```

Comment:
> The entry in the Awesome list should:
Include a short description about the project/theme of the list. It should not describe the list itself.

It should succinctly describe what GitHub Actions is (not the list).

Reply:
Done, but it might be a little too long for your tastes. Let me know if it needs amending. Hard to explain actions in a way that makes sense for people unfamiliar without at least a sentence.